functional/testHeadlessComponents/playlist.xml

```diff
@@ -15,41 +15,10 @@
 <playlist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../TKG/resources/playlist.xsd">
 	<test>
 		<testCaseName>TestHeadlessComponents</testCaseName>
-		<disables>
-			<disable>
-				<comment>https://github.com/adoptium/aqa-tests/issues/5224</comment>
-				<impl>openj9</impl>
-			</disable>
-			<disable>
-				<comment>https://github.com/adoptium/aqa-tests/issues/5224</comment>
-				<impl>ibm</impl>
-			</disable>
-			<disable>
```

Comment:
Can we enable `<impl>hotspot</impl>` only before it's been verified by other vendor?